integration_tests/e2e/induction/createCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -48,7 +48,6 @@ context(`Change links on the Check Your Answers page when creating an Induction`
     // Change in-prison training interests
     Page.verifyOnPage(CheckYourAnswersPage)
       .clickInPrisonTrainingInterestsChangeLink()
-      .hasBackLinkTo(`/prisoners/${prisonNumber}/create-induction/check-your-answers`)
```

Comment:
Because the back link is now a JS back link (`window.history.back()`) we cannot assert what the back link would be / where it would take you.
(And TBH, these assertions were only there to help prove the implementation of the complex rules re: tracking the previous page etc)